### ISSUE TITLE
Avalonia.OpenGL.OpenGlControlBase missing from Avalonia package

### ISSUE THREAD

**Issue by devel0** [AvaloniaUI/Avalonia#4148]
Currently Avalonia.OpenGL 0.9.10 [package](https://www.nuget.org/packages/Avalonia/) includes Avalonia.OpenGL but miss Avalonia.OpenGL.OpenGlControlBase

I was able to find in 0.9.999 from git source build, OpenGlControlBase may a recent addition, of course using these nupkg still report for a missing Avalonia.Native ( I'm building from linux ) with follow result

```sh
devel0@tuf:~/opensource/netcore-sci/examples/0002$ dotnet build
Microsoft (R) Build Engine version 16.5.0+d4cbfca49 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  Restore completed in 30.09 ms for /home/devel0/Documents/opensource/netcore-sci/netcore-sci/netcore-sci.csproj.
  Restore completed in 30.07 ms for /home/devel0/Documents/opensource/netcore-sci/thirdy-clipper/thirdy-clipper.csproj.
/home/devel0/Documents/opensource/netcore-sci/examples/0002/0002.csproj : error NU1102: Unable to find package Avalonia.Native with version (>= 0.9.999)
/home/devel0/Documents/opensource/netcore-sci/examples/0002/0002.csproj : error NU1102:   - Found 32 version(s) in nuget.org [ Nearest version: 0.9.10 ]
/home/devel0/Documents/opensource/netcore-sci/examples/0002/0002.csproj : error NU1102:   - Found 0 version(s) in AvaloniaGit
  Restore failed in 1.45 sec for /home/devel0/Documents/opensource/netcore-sci/examples/0002/0002.csproj.
```

I managed to include Avalonia.Native through [this](https://github.com/SearchAThing-forks/Avalonia/commit/ea73ba1aef167d0ec4336dbf7a4c1c7a7faea9dc) and now the [example](https://github.com/devel0/netcore-sci/tree/master/examples/0002) work; I can make PR for that if considered useful.

**Comment by kekekeks** [comment]
OpenGL control is not available in 0.9.x branch

**Comment by devel0** [comment]
@kekekeks I added a PR #4305 with some improvements to available gl api;
actually control catalog (netcore) seems broken to me after PR #4302 btw I can run OpenGL control reverting those changes. Another example available [here](https://github.com/devel0/netcore-sci/tree/d23959d0fba177382cd23241a6cc51a3ca3356c3#0006)
